src/reports/library/reports.cpp

```diff
@@ -0,0 +1,176 @@
+#include <bbp/sonata/reports.h>
+#include <iostream>
+#include <reports/library/sonatareport.hpp>
+#include <reports/utils/logger.hpp>
+
+using namespace bbp::sonata;
+
+SonataReport sonata_report;
+
+int sonata_clear() {
+    sonata_report.clear();
+    return 0;
+}
+
+int sonata_create_report(
```

Comment:
what is exactly the semantic of the return value? why do you return 0 in any case? shouldn't you return `!= 0` if you are catching an exception?